app/helpers/communicart_mailer_helper.rb

```diff
@@ -14,12 +14,12 @@ def generate_bookend_class(index, count)
     end
   end
 
-  def approve_all_url(approval)
+  def approve_all_url(approval, approve = true)
```

Comment:
I had it that way originally. Problem is that we end up with two methods with 75% identical code, or another layer of helper indirection. I don't feel immensely strongly about it, but I'm not sure either is an improvement over what's there.